hudsonrock/table_hudsonrock_domain_search.go

```diff
@@ -0,0 +1,65 @@
+package hudsonrock
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-hudsonrock/api"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+	"github.com/turbot/steampipe-plugin-sdk/v5/query_cache"
+)
+
+func tableHudsonrockDomainSearch(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "hudsonrock_domain_search",
+		Description: "Search for domain-related cybercrime and infostealer intelligence using Hudson Rock's API.",
+		List: &plugin.ListConfig{
+			KeyColumns: plugin.KeyColumnSlice{
+				{Name: "domain", Require: plugin.Required, CacheMatch: query_cache.CacheMatchExact},
```

Comment:
```suggestion
				{Name: "domain", Require: plugin.Required},
```

We don't need `CacheMatchExact` for required quals. Please remove the `CacheMatchExact` in other table as well.